src/mopidy/internal/deps.py

```diff
@@ -136,9 +136,9 @@ def gstreamer_info() -> DepInfo:
 
     found_elements = []
     missing_elements = []
-    for name, status in _gstreamer_check_elements():
-        if status:
-            found_elements.append(name)
+    for name, version in _gstreamer_check_elements():
+        if version:
+            found_elements.append(f"{name}: {version}")
```

Comment:
Maybe add two-tuples of name and versions to the list here and do the formatting together with the rest of the formatting a bit further down?